tmol/tests/conftest.py

```diff
@@ -26,6 +26,8 @@
     water_box_system,
     water_box_res,
     systems_bysize,
+    pertuzumab_lines,
+    pert_and_nearby_erbb2,
```

Comment:
rather than give ids of the proteins, maybe give something more inline with the aspect they are testing? (antibody, antibody_and_nearby_antigen). 'pert_and_nearby_erbb2' does not mean much to me

Reply:
I think as a name for a fixture, it makes sense to say "what is this?" and not "what is it good for in terms of testing." The unit tests that rely on these fixtures, though, should say "here is what I will be testing", e.g. "dslf-detection" or "chopped-up-chain-representation"

Reply:
I renamed these "pertuzumab_pdb" and "pertuzumab_and_nearby_erbb2_pdb_and_segments"